CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 # Change Log
 
-## [v0.15.9](https://github.com/chef/chef-dk/tree/v0.15.9) (2016-06-10)
-[Full Changelog](https://github.com/chef/chef-dk/compare/v0.15.8...v0.15.9)
+## [v0.15.15](https://github.com/chef/chef-dk/tree/v0.15.15) (2016-06-17)
+[Full Changelog](https://github.com/chef/chef-dk/compare/v0.15.14...v0.15.15)
```

Comment:
Maybe this should be the diff between the last stable build since we're really talking about new things since the last stable build?